brain/neuron/morphologyImpl.h

```diff
@@ -66,7 +69,11 @@ class Morphology::Impl
 
     const uint32_ts& getChildren(const uint32_t sectionID) const;
 
+    const AABB& getBoundingBox() const;
+
 private:
+    std::mutex _mutex;
```

Comment:
unused?

Reply:
Yes, I replaced it with a std::call_once